IO/ConsoleUtil.cs

```diff
@@ -51,7 +51,7 @@ public static void WriteLine(string content, ConsoleColor color = ConsoleColor.W
             lock (_lck)
             {
                 content = content.Trim().Trim(ToTrim);
-                content = string.IsNullOrEmpty(content) ? "" : $"[{DateTime.Now:yyyy-MM-dd HH:mm:ss.fff zzz}] {content}";
+                content = string.IsNullOrEmpty(content) ? string.Empty : $"[{DateTimeOffset.Now:yyyy-MM-dd HH:mm:ss.fff zzz}] {content}";
```

Comment:
DateTimeOffset returns Utc time iirc, not the local one

Reply:
No, `DateTimeOffset.Now` returns local time -> [MSDN](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.now?view=netcore-3.1#System_DateTimeOffset_Now). This is done for micro optimization of memory allocation, the structure is placed only on the stack and doesn't escape into the managed heap. It is used only once and I see no reason to use `DateTime` here.

Reply:
DateTime is also a struct, so this makes no difference

Reply:
OOF, for some reason, I didn't really notice this

Reply:
Fixed in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/b7ca36e4d6727c654dfbf7fe358aee9c53c32f4b